### ISSUE TITLE
Plugin installation was unsuccessful due to error - cannot find module

### ISSUE THREAD

**Issue by vostrom** [floragunncom/search-guard-kibana-plugin#172]
Hi!
I am trying to install the search-guard-plugin for Kibana (version 6.6 on elasticsearch version 6.6 on linux).
It looks like I am missing node modules and the installation fails every time on a different missing module. 
I use `npm install {}` for the missing module and the next one is missing (I have installed requireFrom, moment, https-proxy-agent, etc.). 
Now I am missing '@elastic' module and I cant find it (no traceback or screenshot is added because I am developing in a closed offline network).

Does anyone know a reason for missing node_modules on a plugin installation?

**Comment by jochenkressin** [comment]
How exactly did you install the plugin? Did you use the Kibana plugin install command?

**Comment by vostrom** [comment]
> How exactly did you install the plugin? Did you use the Kibana plugin install command?

Yes I used "./bin/kibana-plugin install file:///tmp/search-guard-kibana-6.6.0-18.zip"

**Comment by jochenkressin** [comment]
I've never encountered this issue before, normally there should be no need to install any node modules manually. Do you see anything in the Kibana logs while installing the plugin? Maybe it's a file-system permission issue? Could you check the permissions on the Search Guard plugin folder?

@mgustafsson1 Any idea from your side?

**Comment by floragunncom** [comment]
@vostrom From where did you download/install Kibana? Did you change anything (or built it yourself)?

**Comment by wcfCode** [comment]
https://search.maven.org/artifact/com.floragunn/search-guard-kibana-plugin/6.6.0-18.1/pom
